docs/book/src/topics/identities.md

```diff
@@ -5,18 +5,12 @@ Also see relevant [identities use cases](identities-use-cases.md), [Azure Active
 
 ## Deprecated Identity Types
 
-<aside class="note warning">
-
-<h1> Warning </h1>
-The capability to set credentials using environment variables has been removed, the required approach is to use `AzureClusterIdentity` as shown with the below supported identity examples.
-</aside>
-
 <aside class="note warning">
 <h1> Warning </h1>
-All of the methods which utilize AAD Pod Identity will no longer function starting in the 1.13 release of CAPZ.
```

Comment:
This message seems worth keeping to me so users know for sure that this is a thing that used to exist but no longer does.